src/Version.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace eLife\ApiClient;
+
+use PackageVersions\Versions;
+
+final class Version
+{
+    private static $version;
+
+    private function __construct()
+    {
+    }
+
+    public static function get() : string
```

Comment:
a smoke test over this (regexp over its return value) could be useful to better pin down what it can produce